dspace-spring-rest/src/main/java/org/dspace/app/rest/BitstreamContentRestController.java

```diff
@@ -37,15 +37,17 @@
 import org.springframework.web.bind.annotation.RestController;
 
 /**
- * This is a specialized controller to provide access to the bitstream binary content
+ * This is a specialized controller to provide access to the bitstream binary
+ * content
  *
  * @author Andrea Bollini (andrea.bollini at 4science.it)
  * @author Tom Desair (tom dot desair at atmire dot com)
  * @author Frederic Van Reet (frederic dot vanreet at atmire dot com)
  *
  */
 @RestController
-@RequestMapping("/api/"+BitstreamRest.CATEGORY +"/"+ BitstreamRest.PLURAL_NAME + "/{uuid:[0-9a-fxA-FX]{8}-[0-9a-fxA-FX]{4}-[0-9a-fxA-FX]{4}-[0-9a-fxA-FX]{4}-[0-9a-fxA-FX]{12}}/content")
+@RequestMapping("/api/" + BitstreamRest.CATEGORY + "/" + BitstreamRest.PLURAL_NAME
+		+ "/{uuid:[0-9a-fxA-FX]{8}-[0-9a-fxA-FX]{4}-[0-9a-fxA-FX]{4}-[0-9a-fxA-FX]{4}-[0-9a-fxA-FX]{12}}/content")
```

Comment:
As noted in #1889, I'd like to see a comment that explains these very large regex.  It could be an example of paths that should match here, or note that it's trying to match valid UUIDs (if that's what this is doing).  It's not a very straightforward regex, so it needs comments.

Reply:
yes please add a comment as suggested by @tdonohue 
The requestMapping value matches valid string representation of an UUID